test/com/puppetlabs/puppetdb/test/scf/storage.clj

```diff
@@ -510,6 +627,8 @@
     (is (= (query-to-vec ["SELECT COUNT(*) as c FROM catalog_resources"])
            [{:c 3}]))))
 
+;; FIXME: should this and delete-certname-facts! go away once the new
```

Comment:
@senior  Yes, and to anyone reviewing.  I'll deal with it one way or another before the final version, but am happy to hear other opinions.